doc/source/tune/_tutorials/tune-xgboost.rst

```diff
@@ -0,0 +1,510 @@
+.. _tune-xgboost:
+
+Tuning XGBoost parameters
+=========================
+
+XGBoost is currently one of the most popular machine learning algorithms. It performs
+very well on a large selection of tasks, and was the key to success in many Kaggle
+competitions.
+
+This tutorial will give you a quick introduction to XGBoost, show you how
+to train an XGBoost model, and then guide you on how to optimize XGBoost
+parameters using Tune to get the best performance. We tackle the following topics:
+
+.. contents:: Table of contents
+   :depth: 2
+
+.. note::
+
+    To run this tutorial, you will need to install the following:
+
+    .. code-block:: bash
+
+        $ pip install xgboost
+
+What is XGBoost
+---------------
```

Comment:
Is it possible to shorten this section? I think you can cut the region on boosting/bagging/other libraries, and just link readers to another blog post or the XGBoost website.

Reply:
I made it quite a bit shorter and referred to another overview article